README.md

```diff
@@ -1,6 +1,6 @@
 # babel-plugin-react-transform
 
-This Babel plugin wraps all React components into arbitrary transforms written by the community.  
```

Comment:
Ah sorry, I thought I had my trailing spaces plugin set up to not act on markdown, but I appear to have screwed it up.

The other PR for docs is better anyway!